packages/@aws-cdk/aws-apigateway/lib/rest-api-base.ts

```diff
@@ -273,6 +273,13 @@ export abstract class RestApiBase extends Resource implements IRestApi {
     return this._domainName;
```

Comment:
Please revert this file rename.

Reply:
Hi,
Yes, I was trying to create metrics for RestApi.
I'm a little bit confused here... When should metrics be declared in the source code and when via cfn augmentations? For apigatewayv2 and HTTP Api specifically, what would be the correct approach?
Thanks

Reply:
The correct approach would be to add these in code.

Reply:
Added to code